code/game/objects/items.dm

```diff
@@ -744,24 +744,26 @@ modules/mob/living/carbon/human/life.dm if you die, you will be zoomed out.
 					return
 
 			if(85 to 93)
-				user << SPAN_DANGER("Your [src] broke beyond repair!")
```

Comment:
It was possible for robots to break their modules in tool failures. Ishuman checks added to prevent this